repo/datastore.go

```diff
@@ -431,3 +432,17 @@ type WatchedScriptStore interface {
 	Queryable
 	wallet.WatchedScripts
 }
+
+// MessageStore is the messages table interface
+type MessageStore interface {
+	Queryable
+
+	// Save a new message
+	Put(messageID, orderID string, mType pb.Message_MessageType, peerID string, msg Message) error
+
+	// GetByOrderIDType returns the message for specified order and type
+	GetByOrderIDType(orderID string, mType pb.Message_MessageType) (*Message, string, error)
+
+	// GetByMessageID returns the message for specified message id
+	GetByMessageID(messageID string) (*Message, string, error)
```

Comment:
This does not appear to be used at all? If not, let's remove this and the associated indices.

Reply:
This method has been removed. The indices' future use has been discussed. It is beeter to have an index now when the table is being created rather than adding an index later which may cause issues.

Reply:
I'm not sure what issues might arise from creating an index later (aside from the extra work of handling the migration, which I agree is annoying). Thanks for the explanation. Let's leave the index in place.